inductiva/api/methods.py

```diff
@@ -122,13 +123,12 @@ def download_output(
     Return:
         Downloads and extracts the data to the client.
     """
-    logging.info("Downloading output...")
+    logging.info("Downloading the outputs of the task %s...", task_id)
```

Comment:
Maybe show information to where it is being downloaded. I know that we show the "target" folder after downloading. But, if the task is super heavy and has many files it would be cool to see the folder to which it is being downloaded